[2025-08-03T10:49:08.353+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:48:00+00:00/task_id=validate_model_performance permission to 509
[2025-08-03T10:49:08.354+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:48:00+00:00/task_id=validate_model_performance permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:48:00+00:00/task_id=validate_model_performance'
[2025-08-03T10:49:08.395+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:48:00+00:00/task_id=validate_model_performance permission to 509
[2025-08-03T10:49:08.397+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:48:00+00:00/task_id=validate_model_performance permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:48:00+00:00/task_id=validate_model_performance'
[2025-08-03T10:49:08.438+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: train_predict_to_db.validate_model_performance scheduled__2025-08-03T10:48:00+00:00 [queued]>
[2025-08-03T10:49:08.449+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: train_predict_to_db.validate_model_performance scheduled__2025-08-03T10:48:00+00:00 [queued]>
[2025-08-03T10:49:08.450+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-08-03T10:49:08.466+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): validate_model_performance> on 2025-08-03 10:48:00+00:00
[2025-08-03T10:49:08.480+0000] {standard_task_runner.py:57} INFO - Started process 904 to run task
[2025-08-03T10:49:08.484+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'train_predict_to_db', 'validate_model_performance', 'scheduled__2025-08-03T10:48:00+00:00', '--job-id', '80', '--raw', '--subdir', 'DAGS_FOLDER/train_predict_to_db_dag.py', '--cfg-path', '/tmp/tmp6z6_dk_0']
[2025-08-03T10:49:08.489+0000] {standard_task_runner.py:85} INFO - Job 80: Subtask validate_model_performance
[2025-08-03T10:49:08.553+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:48:00+00:00/task_id=validate_model_performance permission to 509
[2025-08-03T10:49:08.554+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:48:00+00:00/task_id=validate_model_performance permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:48:00+00:00/task_id=validate_model_performance'
[2025-08-03T10:49:08.557+0000] {task_command.py:416} INFO - Running <TaskInstance: train_predict_to_db.validate_model_performance scheduled__2025-08-03T10:48:00+00:00 [running]> on host 407839faddcf
[2025-08-03T10:49:08.668+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='train_predict_to_db' AIRFLOW_CTX_TASK_ID='validate_model_performance' AIRFLOW_CTX_EXECUTION_DATE='2025-08-03T10:48:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-08-03T10:48:00+00:00'
[2025-08-03T10:49:08.712+0000] {python.py:194} INFO - Done. Returned value was: {'mse': 1708017.7583326078, 'rmse': 1306.9115342411696, 'mae': 950.42332734002, 'r2': 0.9090710593528853}
[2025-08-03T10:49:08.754+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=train_predict_to_db, task_id=validate_model_performance, execution_date=20250803T104800, start_date=20250803T104908, end_date=20250803T104908
[2025-08-03T10:49:08.818+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-08-03T10:49:08.839+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
